src/os/KeyValueStore.cc

```diff
@@ -530,10 +530,6 @@ KeyValueStore::KeyValueStore(const std::string &base,
   op_queue_len(0), op_queue_bytes(0),
   op_throttle_lock("KeyValueStore::op_throttle_lock"),
   op_finisher(g_ceph_context),
-  op_tp(g_ceph_context, "KeyValueStore::op_tp",
```

Comment:
Does this call for removal of configurable values keyvaluestore_op_threads and keyvaluestore_op_thread_suicide_timeout from config_opts.h?